R/ui.R

```diff
@@ -85,6 +86,11 @@ include_scripts <- function() {
     "js/trigger-button.js",
     "js/navigate-away.js"
```

Comment:
Instead of the code below, you can just put `opt$additional_scripts` inside the first definition of `scripts`, and `c()` should do the job for you.